templates/httpd/config/httpd.conf

```diff
@@ -0,0 +1,550 @@
+#
```

Comment:
for use the generic template functions it expects the files in templates/cr.Kind/config